docs/home/howto/DATABASES.md

```diff
@@ -31,6 +31,19 @@ specification:
 If enable is set to yes in replication then Epiphany will automatically create cluster of master and slave server with replication user with name and password
 specified in data.yaml.
 
+## How to setup Postgresql connection pooling
+
+Postgresql connection pooling in Epiphany is served by PGBouncer application. This might be added as a feature if needed. Simplest configuration run PGBouncer on Postgresql master node. This needs to be enabled in configuration yaml file:
```

Comment:
```suggestion
Postgresql connection pooling in Epiphany is served by PGBouncer application. This might be added as a feature if needed. Simplest configuration runs PGBouncer on Postgresql master node. This needs to be enabled in configuration yaml file:
```